### ISSUE TITLE
I'm getting error when trying to use plugin in podfile

### ISSUE THREAD

**Issue by Vitalii-Gozhenko** [joncardasis/cocoapods-user-defined-build-types#1]
```
[!] Invalid `Podfile` file: uninitialized constant Pod::Target::BuildType
Did you mean?  Pod::BuildType.
```

**Comment by ebgraham** [comment]
same, on this line
```
  pod 'React', :path => '../node_modules/react-native/', :build_type => :dynamic_framework
```

**Comment by dudihisine** [comment]
You can Fix it by going to this folder:
`/Library/Ruby/Gems/2.6.0/gems/cocoapods-user-defined-build-types-0.0.6/lib/cocoapods-user-defined-build-types`

copy the "podfile_options.rb" file to your desktop

change the original file name to "podfile_options_old.rb"

edit the copied file by change the:
`Pod::Target::BuildType`
to:
`Pod::BuildType`

save the file.
copy&paste it to the folder

you will need Admin permissions so probably you will enter your Mac password couple of times.

**Comment by ghost** [comment]
Could be due to this issue https://github.com/joncardasis/cocoapods-user-defined-build-types/issues/3

**Comment by dudihisine** [comment]
Probably, the in version 1.9.1 Cocoapods changed it from  'Pod::Target::BuildType' to 'Pod::BuildType'

**Comment by joncardasis** [comment]
@dudihisine you're right on the money. Looks like the namespace updated in 1.9.1. I'll begin working on a fix for this.

**Comment by joncardasis** [comment]
This is now fixed in v0.0.7. See https://github.com/joncardasis/cocoapods-user-defined-build-types/commit/c9077a02f2b79e208b84a1da13a048131f5c92d6.

**Comment by gelinxiao** [comment]
pod version 1.5.3:  has same error
